blockpage.js

```diff
@@ -49,43 +49,33 @@ function fetchProtectionRules(url,status){
   fetch(url)
   .then((res) => res.json())
   .then((domains) => {
+    for (var i=0; i<domains.length; i++){
+      domains[i]="*://www."+domains[i]+"/*"
```

Comment:
It should not rely on www, www is not as widely used as it was in the past as it is no longer necessary